main/proxy.go

```diff
@@ -20,13 +20,17 @@ import (
 	"archive/zip"
 	"bytes"
 	"flag"
+	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
 
 	"github.com/apache/incubator-openwhisk-runtime-go/openwhisk"
 )
 
+// flag to enable debug
```

Comment:
Comment should be "flag to display version" or similar

Reply:
done